src/plonk/mod.rs

```diff
@@ -154,6 +154,14 @@ pub struct PlonkStructure<F: PrimeField> {
 
     pub(crate) custom_gates_lookup_compressed: CompressedGates<F>,
 
+    /// TODO: after we switch from Sangaria IVC to IVC with cyclefold + protogalaxy
```

Comment:
All "TODO's" must be accompanied by an issue number, because without that - we just forgot about it.

Reply:
We have auto TODO issue creation in other repos